app/assets/javascripts/discourse/app/widgets/post.js

```diff
@@ -41,6 +43,19 @@ function transformWithCallbacks(post, topicUrl, store) {
   return transformed;
 }
 
+let postMenuWidgetExtensionsAdded = null;
+let postMenuConsoleWarningLogged = false;
+
+registerDeprecationHandler((_, opts) => {
```

Comment:
Since we added a global silence for the deprecation... I don't think this handler will ever fire?

I guess that's ok. It just means that the 'auto' mode won't work properly until we remove that global silence.

Should we unsilence globally before merging this PR? Or you prefer to do it later?